onnxruntime/contrib_ops/cpu/quantization/matmul_nbits.cc

```diff
@@ -46,15 +47,17 @@ Status MatMulNBits::Compute(OpKernelContext* ctx) const {
   auto status = ctx->GetTempSpaceAllocator(&allocator);
   ORT_RETURN_IF_ERROR(status);
   auto tmp_b_data_ptr = IAllocator::MakeUniquePtr<float>(allocator, SafeInt<size_t>(K_) * N_);
-  DequantizeBlockwise<float>(tmp_b_data_ptr.get(),
-                             b_data,
-                             scales_data,
-                             zero_points_data,
-                             static_cast<int32_t>(block_size_),
-                             static_cast<int32_t>(nbits_),
-                             static_cast<int32_t>(N_),
-                             static_cast<int32_t>(K_),
-                             thread_pool);
+
```

Comment:
Don't we need nbits input or template argument for future extension?

Reply:
bits expansion is not that easy, we need validation and all sorts of work done first, no rush.

Reply:
allowing for it in the interface would be good, even if it only supports 4 bit for now.
alternatively, hardcoding 4 bit in the name as in `DequantizeBlockwise4b` would at least be explicit about it.

I'm trying to use this function in places where the bit width is a parameter so being able to pass that through would be nice.

Reply:
fixed with new commit